packages/moonstone/Scrollable/Scrollable.js

```diff
@@ -73,6 +75,45 @@ const configureSpotlightContainer = ({'data-spotlight-id': spotlightId, focusabl
 	});
 };
 
+/*
+ * Track the last position of the pointer to check if a list should scroll by
+ * page up/down keys when the pointer is on a list without any focused node.
+ * `keydown` event does not occur if there is no focus on the node and
+ * its descendants, we add `keydown` handler to `document` also.
+ */
+const
+	lastPointer = {x: 0, y: 0},
+	pointerTracker = (ev) => {
+		lastPointer.x = ev.clientX;
+		lastPointer.y = ev.clientY;
+	};
+
+const
+	// An app could have lists and/or scrollers more than one,
+	// so we should test all of them when page up/down key is pressed.
+	scrollables = new Set(),
+	pageKeyHandler = (ev) => {
+		if (Spotlight.getPointerMode() && !Spotlight.getCurrent()) {
```

Comment:
It might be best to include the keycode checks here to avoid the potentially expensive `getElementFromPoint` call for keys we don't care about.

Reply:
Updated!